src/reducers/query.jsx

```diff
@@ -53,7 +68,20 @@ export function toggleFilter(state, action) {
   }
 }
 
-// TODO: Set defaultQueryState to recognize existing URL params
+function removeFilter(state, action) {
```

Comment:
We can refactor for consistency.  I like the idea of a toggle action so I want to keep it.  Maybe we just figure out how to name it better